.github/workflows/pr-checks-for-qa-approvals.yml

```diff
@@ -0,0 +1,66 @@
+
+#
+# Checks if QA approvals are necessary, and if so runs the associated action when a new review is added
+#
+
+name: '[Utils] QA required approval'
+
+on:
+  pull_request:
+    types: [opened, edited, converted_to_draft, ready_for_review, reopened]
+    paths:
+      - 'VAMobile/src/**'
+      - 'VAMobile/package.json'
+      - '!VAMobile/src/**.test.*'
+  pull_request_review:
+    types: [submitted, dismissed]
+  pull_request_target:
+    branches:
+      - 'develop'
```

Comment:
We'll probably want to include the `paths` option here as well, to prevent this from running on irrelevant PRs that have their head branch updated

Reply:
Are referring to these path theo?
 - 'VAMobile/src/**'
      - 'VAMobile/package.json'
      - '!VAMobile/src/**.test.*'
if yes, I included right before 'on pull_request_review"